lib/arkana/templates/arkana.swift.erb

```diff
@@ -5,7 +5,14 @@
 // Automatically generated by Arkana (https://github.com/rogerluan/arkana)
 
 import Foundation
+<% if @should_cocoapods_cross_import_modules %>
```

Comment:
I believe the current behavior (prior to this PR, in `main`) should remain the default, to avoid regressions, no?

Reply:
Is this logic you implemented correct? Just a sanity check...

### `should_cocoapods_cross_import_modules == true` (default)

Output:

```swift
#if COCOAPODS
#else
import MyArkanaInterfaces
#endif
```

### `should_cocoapods_cross_import_modules == false`

Output

```swift
import MyArkanaInterfaces
```

----

I believe the desired logic would be:

Default:

```swift
import MyArkanaInterfaces
```

Not-default:

Either `nothing` (i.e. don't import MyArkanaInterfaces and don't do anything), or:

```swift
#if COCOAPODS
#else
import MyArkanaInterfaces
#endif
```

To be honest, I don't know whether that preprocessor macro will work in all scenarios, will it?

Reply:
And this macro will work when using Cocoapods. The COCOAPODS flag is automatically added when using `pod install`.